plugin/api.go

```diff
@@ -249,6 +249,9 @@ type API interface {
 	// Minimum server version: 5.3
 	GetFileInfo(fileId string) (*model.FileInfo, *model.AppError)
 
+	// GetFileThumbnail gets a file's thumbnail for a specific fileId.
```

Comment:
Can you document the minimum version required here?

```suggestion
	// UploadFile will upload a file to a channel using a multipart request, to be later attached to a post.
	//
	// Minimum server version: 5.6
```